library/experiments/hate_speech.py

```diff
@@ -61,14 +61,14 @@
 
 huggingface_baseline = create_nlp_huggingface_pipeline(
     title="hf-distillbert",
-    input=input_data,
+    input=tweet_eval_hate,
```

Comment:
ohh shit! now I realized why we didn't have this before.
Previously, I could easily swap out the dataset for an experiment. This is what we were actually doing further down. I have various Experiments that are generated by replacing the input type with a different one.